snyk-operator-certified/Makefile

```diff
@@ -1,5 +1,5 @@
 # Current Operator version
-VERSION ?= 1.64.1
+VERSION ?= 1.67.2
```

Comment:
What does the `?` here do? Is it like `>=`?

Reply:
Conditional assignment - if the VERSION variable is not already set, it will set it to that value `1.67.2`